doc/assets/cover.tmpl

```diff
@@ -1,16 +1,28 @@
+.. class:: logoimage
```

Comment:
Are we updating the cover due to the inkscape/pdfrw change? Or is this a different thing?

Reply:
The cover had a background image pdf that depended on pdfrw to work. As I had to change that image anyway, I decided to put in the logo and fix the look of the title.